images/linux/scripts/tests/Tools.Tests.ps1

```diff
@@ -119,3 +119,121 @@ Describe "gfortran" {
         "$GfortranVersion --version" | Should -ReturnZeroExitCode
     }
 }
+
+Describe "Git" {
+    It "git" {
+        "git --version" | Should -ReturnZeroExitCode
+    }
+
+    It "git-lfs" {
+        "git-lfs --version" | Should -ReturnZeroExitCode
+    }
+
+    It "git-ftp" {
+        "git-ftp --version" | Should -ReturnZeroExitCode
+    }
+
+    It "hub-cli" {
+        "hub --version" | Should -ReturnZeroExitCode
+    }
+}
+
+Describe "Heroku" {
+    It "heroku" {
+        "heroku --version" | Should -ReturnZeroExitCode
+    }
+}
+
+Describe "HHVM" {
+    It "hhvm" {
+        "hhvm --version" | Should -ReturnZeroExitCode
+    }
+}
+
+Describe "Homebrew" {
+    It "homebrew" {
+        "brew --version" | Should -ReturnZeroExitCode
+    }
+}
+
+Describe "Julia" {
+    It "julia" {
+        "julia --version" | Should -ReturnZeroExitCode
+    }
+}
+
+Describe "Kubernetes tools" {
```

Comment:
nitpick: avoid spaces `KubernetesTools`

Reply:
But why? We have a lot of such Describes with witespaces even for ubuntu:
https://github.com/actions/virtual-environments/blob/6245d40db722e18bc79096e52ae02f29838e0cf5/images/linux/scripts/tests/Tools.Tests.ps1#L43
https://github.com/actions/virtual-environments/blob/6245d40db722e18bc79096e52ae02f29838e0cf5/images/linux/scripts/tests/CLI.Tools.Tests.ps1#L2
https://github.com/actions/virtual-environments/blob/6245d40db722e18bc79096e52ae02f29838e0cf5/images/linux/scripts/tests/CLI.Tools.Tests.ps1#L14
and so on